test/Unit/Durable/DurableControllerTests.cs

```diff
@@ -133,70 +136,52 @@ public void TryGetInputBindingParameterValue_RetrievesOrchestrationContextParame
-
+            var inputData = new[] { CreateParameterBinding(_contextParameterName, _orchestrationContext) };
             var durableController = CreateDurableController(DurableFunctionType.OrchestrationFunction);
 
-            _mockPowerShellServices.Setup(_ => _.SetOrchestrationContext(It.IsAny<OrchestrationContext>()));
-            durableController.BeforeFunctionInvocation(inputData);
+            _mockPowerShellServices.Setup(_ => _.SetOrchestrationContext(
+                It.IsAny<ParameterBinding>(),
+                out It.Ref<IExternalOrchestrationInvoker>.IsAny))
+            .Returns(_orchestrationBindingInfo);
+            _mockOrchestrationInvoker.Setup(_ => _.SetExternalInvoker(It.IsAny<IExternalOrchestrationInvoker>()));
+            durableController.InitializeBindings(inputData);
 
             var expectedResult = new Hashtable();
             _mockOrchestrationInvoker.Setup(
                 _ => _.Invoke(It.IsAny<OrchestrationBindingInfo>(), It.IsAny<IPowerShellServices>()))
                 .Returns(expectedResult);
 
-            var invoked = durableController.TryInvokeOrchestrationFunction(out var actualResult);
-            Assert.True(invoked);
+            var actualResult = durableController.InvokeOrchestrationFunction();
             Assert.Same(expectedResult, actualResult);
 
             _mockOrchestrationInvoker.Verify(
                 _ => _.Invoke(
                     It.Is<OrchestrationBindingInfo>(
-                        bindingInfo => bindingInfo.Context.InstanceId == orchestrationContext.InstanceId
-                                       && bindingInfo.ParameterName == contextParameterName),
+                        bindingInfo => bindingInfo.Context.InstanceId == _orchestrationContext.InstanceId
+                                       && bindingInfo.ParameterName == _contextParameterName),
                     _mockPowerShellServices.Object),
                 Times.Once);
         }
 
-        [Theory]
-        [InlineData(DurableFunctionType.None)]
-        [InlineData(DurableFunctionType.ActivityFunction)]
-        internal void TryInvokeOrchestrationFunction_DoesNotInvokeNonOrchestrationFunction(DurableFunctionType durableFunctionType)
-        {
-            var contextParameterName = "ParameterName";
-            var orchestrationContext = new OrchestrationContext { InstanceId = Guid.NewGuid().ToString() };
-            var inputData = new[] { CreateParameterBinding(contextParameterName, orchestrationContext) };
-
-            var durableController = CreateDurableController(durableFunctionType);
-
-            _mockPowerShellServices.Setup(_ => _.SetOrchestrationContext(It.IsAny<OrchestrationContext>()));
-            durableController.BeforeFunctionInvocation(inputData);
-
-            var invoked = durableController.TryInvokeOrchestrationFunction(out var actualResult);
-            Assert.False(invoked);
-            Assert.Null(actualResult);
-        }
-
```

Comment:
We don't have this method anymore because of how the logic has been refactored. We now guarantee that the orchestration invoker is only called for orchestrator types, instead of always calling it and requiring `TryInvokeOrchestratorFunction` to do the check for us.